packages/ui-extensions/docs/surfaces/admin/build-docs.sh

```diff
@@ -10,6 +10,15 @@ fail_and_exit() {
   exit $1
 }
 
+run_sed() {
+  if [ -z $SPIN ]
+  then
+    sed -i '' "$1" $2
```

Comment:
According to chatGPT: The '' after -i is used for compatibility with macOS's version of sed, which requires an argument for -i (the empty string indicates no backup should be made).

Is this correct and can you explain this a bit more? Why does Spin need the MacOS version and why no back up on spin? Is there a backup when it runs locally?

Reply:
Oh I just read your comment in the PR description ("idk why"). 👌